src/kp2bw/bw_types.py

```
@@ -0,0 +1,177 @@
+]
+
+
+# ---------------------------------------------------------------------------
+# Concrete URI and field types — stricter than the generated NotRequired versions
+# ---------------------------------------------------------------------------
+
+
+class BwUri(TypedDict):
+    """Single URI entry on a login item.
+
+    The spec types ``match`` as ``Literal[0–5]``; ``None`` is also valid and
```

Ruff (0.15.2)

[warning] Docstring contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF002)